src/System.IO.MemoryMappedFiles/src/Interop/Interop.Windows.cs

```diff
@@ -211,6 +210,12 @@ internal static partial class mincore
             ref SECURITY_ATTRIBUTES lpFileMappingAttributes, int flProtect,
             int dwMaximumSizeHigh, int dwMaximumSizeLow, string lpName);
 
+        [DllImport(MEMORYDLL, EntryPoint = "CreateFileMappingW", CharSet = CharSet.Unicode, SetLastError = true)]
+        [SecurityCritical]
+        internal static extern SafeMemoryMappedFileHandle CreateFileMapping(IntPtr hFile,
```

Comment:
Can the hFile parameter here be some type of SafeHandle?  I think that it would imply INVALID_HANDLE_VALUE would have to change type, but I think that's ok?